site/profile/manifests/core/telegraf.pp

```diff
@@ -1,61 +0,0 @@
-# @summary
```

Comment:
Does the removal of this profile mean that we no longer need telegraf to monitor power at the summit and that `hieradata/node/gs-telegraf-node-01.cp.lsst.org.yaml` may also be deleted?  Can that node be retried too?

Reply:
Already deleted :)